app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+from os import path, makedirs
+from shutil import move
+
+
+def move_file(command: str) -> None:
+    _, source, destination = command.split()
+
+    if (not (command.startswith("mv "))
```

Comment:
Why don't you get this in the line 6, where you split str if you want to check it here?

Reply:
I thought that it would work faster